lib/update-bridgehead.sh

```diff
@@ -4,10 +4,15 @@ source lib/functions.sh
 AUTO_HOUSEKEEPING=${AUTO_HOUSEKEEPING:-true}
 
 if [ "$AUTO_HOUSEKEEPING" == "true" ]; then
-	A="Performing automatic maintenance: Cleaning docker images."
+	A="Performing automatic maintenance: "
+	if bk_is_running; then
+		A="$A Cleaning docker images."
+		docker system prune -a -f
```

Comment:
sollen wir auch gleich ein volume prune ergänzen? (für ungenutzte volumes)

Reply:
Not sure (requires testing). Please make another PR for that